frontend/src/app/libs/sign-in/mock-signin.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { of, Observable } from 'rxjs';
+import { delay } from 'rxjs/operators';
+import { IUser, IRegisteredUser } from '../container/store';
+
+const delay100: number = 100;
+
+@Injectable({
+	providedIn: 'root',
+})
+export class MockSigninService {
+	private user: IRegisteredUser = {
```

Comment:
Why do we need `IRegisteredUser` interface? Seems like it's just user and there are no need to use word `registered`.